rs/tests/consensus/consensus_performance.rs

```diff
@@ -132,7 +132,6 @@ fn setup(env: TestEnv) {
         NnsCanisterWasmStrategy::TakeBuiltFromSources,
         NnsCustomizations::default(),
     );
-    set_authorized_subnets(&env);
```

Comment:
@eichhorl Do you know/remember why this line is here? I removed it and everything seems to work as before

Reply:
As I understand this sends a proposal that, after execution, allows everyone to deploy canisters on every subnet. So it might be necessary if you want to install canisters on the NNS or SNS subnet

Reply:
Thanks. I guess it's not necessary in our case? In the test, we seem to be installing canisters on the application subnets only, and it seems that we can still do it without this line.